pennylane/labs/resource_estimation/resource_tracking.py

```diff
@@ -71,6 +81,11 @@
     "precision_qubit_unitary": 1e-9,
     "precision_qrom_state_prep": 1e-9,
 }
+r"""A configuration dictionary which stores the default values of certain parameters
+(e.g. error thresholds). The default ``config`` argument used in the ``estimate_resources()``
+function. This allows users to have top level access to change the default value of certain
+parameters if none were provided in the circuit.
```

Comment:
Again, a bit unclear to me how users can change the default. By copying this set and changing the numbers, then passing it to config in estimate_resources?